packages/usa-combo-box/src/styles/_usa-combo-box.scss

```diff
@@ -56,13 +56,23 @@ $close: map-merge(
   margin-bottom: 0;
   max-width: none;
   padding-right: calc(2.5em + 3px);
+
+  &:disabled,
+  &[aria-disabled="true"] {
+    @include u-disabled;
+  }
 }
 
 button.usa-combo-box__toggle-list,
 button.usa-combo-box__clear-input {
   &:focus {
     outline-offset: -4px;
   }
+
+  &:disabled,
+  &[aria-disabled="true"] {
+    @include u-disabled;
+  }
```

Comment:
I see this is set on the toggle list. Is there an example of how this would look?

Reply:
Inspected classes, they're the arrow dropdown and clear icons once component is initialized and option has been selected.

![image](https://user-images.githubusercontent.com/3385219/208716871-e60966d7-f37f-44ee-ac4a-5263d73cf547.png)